rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -153,101 +214,116 @@ components:
+        - id
+        - name
+        - product
+        - version
+        - host
+        - port
+        - context
       properties:
-        remoteId:
+        id:
           type: string
           description: >
             A system advertised identifier that is meant to uniquely identify the system. The
-            identifier is not guaranteed to be unique from Ion's perspective which might be
-            forced to use other properties to uniquely identify the system registering. This
-            identifier is typically generated differently based on the advertised product and/or
-            version.
+            identifier is not guaranteed to be unique from Ion's perspective which might be forced
+            to use other properties to uniquely identify the system registering. This identifier is
+            typically generated differently based on the advertised product and/or version.
           minLength: 1
           example: abcd-01234-xxxx
+        name:
+          type: string
+          description: >
+            The name of the remote system (a.k.a. site name). This name will be displayed to an
+            administrator when showing the remote system.
+          minLength: 1
+          example: Acme Intelligence Node
+        organization:
+          type: string
+          description: >
+            The organization the remote system belongs to.
+          example: Acme Organization
+        contact:
+          type: string
+          description: >
+            The contact information for the remote system.
+          example: support@acme.org
+        product:
+          type: string
+          description: >
+            The type of product the remote system is. This will help Ion determine how to
+            communicate and what functionnalities are available.
+          minLength: 1
+          example: ddf, alliance
+        version:
+          type: string
+          description: >
+            The version of the product installed on the remote system. This  will help Ion
+            determine how to best communicate with the system.
+          minLength: 1
+          example: 4.6.9
+        protocol:
+          type: string
+          description: >
+            The protocol to use when communicating with the remote system.
+          enum:
+            - http
+            - https
+          default: https
+        host:
+          type: string
+          description: >
+            The hostname (or IP address) where the remote system can be reached. This should be an
+            externally accessible hostname.
+          minLength: 1
+          example: myhost.com
+        port:
+          type: integer
+          format: int32
+          description: >
+            The port number where the remote system can be reached. For DDF-based systems, this
+            would be the port number hosting the CSW and GraphQL endpoints.
+          minimum: 0
+          exclusiveMinimum: true
+          maximum: 49152
```

Comment:
❓ Why not up to 65535?

Reply:
because above 49152 are ephemeral ports

Reply:
From what I see that is a suggestion by IANA. Are we sure no one will be using ports in that range?

Reply:
Good question. I am not against opening it up in the spec. I was just trying to clearly state the limits. What do you think @lessarderic?

Reply:
After more reflection, I replaced protocol, host, port, and context into one single url property which takes care of this comment.